src/platform/packages/shared/kbn-unified-data-table/src/components/data_table_additional_display_settings.tsx

```diff
@@ -113,13 +113,13 @@ export const UnifiedDataTableAdditionalDisplaySettings: React.FC<
     settings.push(
       <RowHeightSettings
         rowHeight={rowHeight}
-        rowHeightLines={rowHeightLines}
         label={i18n.translate('unifiedDataTable.rowHeightLabel', {
-          defaultMessage: 'Cell row height',
+          defaultMessage: 'Max body cell lines',
```

Comment:
```suggestion
          defaultMessage: 'Body cell lines',
```
I think we'll need to change this to "Body cell lines" for now. The mockups assume we're using the new max lines behaviour EUI added in https://github.com/elastic/eui/pull/8096, but we haven't enabled it yet. We can start exploring that as a followup, but it needs pretty extensive performance testing so definitely for a separate PR.

Reply:
No problem, will update the issue to reflect this (so there will be no surprises why it's changed in future) and update it here!